sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Clauses/FromClause.cs

```diff
@@ -1,31 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace Azure.DigitalTwins.Core.QueryBuilder
 {
     /// <summary>
     /// Custom object for a FROM clause. Only meant to be used when adding FROM to a query. Hidden from user.
     /// </summary>
-    internal class FromClause : BaseClause
+    internal class FromClause : ClauseBase
     {
         /// <summary>
         /// The collection to query from. Stored in an enum to give user some intellisense since there are only two possible collection types,
         /// ADT instances or relationships.
         /// </summary>
-        public AdtCollection Collection { get; set; }
+        public string Collection { get; private set; }
```

Comment:
properties after ctors

order should be:
* fields
* ctors
* properties
* public methods
* internal methods
* protected methods
* private methods